pkg/controller/operators/olm/operator.go

```diff
@@ -141,6 +148,7 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 		clientAttenuator:      scoped.NewClientAttenuator(config.logger, config.restConfig, config.operatorClient),
 		serviceAccountQuerier: scoped.NewUserDefinedServiceAccountQuerier(config.logger, config.externalClient),
 		clientFactory:         clients.NewFactory(config.restConfig),
+		isCopiedCSVsEnabled:   config.isCopiedCSVsEnabled,
```

Comment:
One other consideration for feature flagging vs. a CLI toggle is that we might be able to forgo expanding the config structure further, and instead just check whether that feature gate has been enabled at runtime.

Reply:
Flagging is not a viable solution to customers that want to disable olm csv copying as users will need to mark OLM as unmanged by CVO, thereby placing their cluster in an unmanaged/unupgradeable state.